src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -196,7 +196,7 @@ class NetworkOPsImp final
     // VFALCO TODO Make LedgerMaster a SharedPtr or a reference.
     //
     NetworkOPsImp (Application& app, NetworkOPs::clock_type& clock,
-        bool standalone, std::size_t network_quorum, bool start_valid,
+        bool standalone, std::size_t minPeerCount, bool start_valid,
```

Comment:
We do want to be consistent. Several files contain an arcane mix of styles. Really old, old old, new old, old new, new new... it's weird.

I'm not a huge fan of going through and renaming _everything_ at once, especially as part of some other fix, and I'd normally leave this well enough alone, except the name was bad to begin with. Since I was fixing it, I decided to also move it to the current preferred style.

I don't feel that strongly about it though and am happy to just rename it `m_min_peer_count`, even if that's way too many underscores for my overworked SHIFT finger.